RocketChat/Rocket.Chat.ReactNative
# [FIX] Jitsi users being muted always

## Proposed changes

Probably some commit after 2.10.2 was cherry picked when the sdk was builded on the last time, now I've set the branch to the specific tag `v.2.10.2`.
I've reverted some commits of `jitsi-meet` and regenerated the sdk, `v.2.10.2`.
- [X] Update [RocketChat/jitsi-meet#mobile](https://github.com/RocketChat/jitsi-meet/tree/mobile)

## Issue(s)	

Close #2622 

## How to test or reproduce

- Start a Jitsi call using two different devices
- Unmute all users
- You should hear yourself

## Screenshots
![image](https://user-images.githubusercontent.com/29778115/98861274-c2877c80-2443-11eb-9109-45cffcc47a87.png)
![Screen Shot 2020-11-11 at 5 30 12 PM](https://user-images.githubusercontent.com/29778115/98861278-c4e9d680-2443-11eb-8d8b-04da3d8d18ab.png)


## Types of changes




## Checklist



## Further comments
